emission/exportdata/export_data.py

```diff
@@ -0,0 +1,38 @@
+# Standard imports 
+import logging
+
+# Our imports
+import emission.storage.pipeline_queries as espq
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.decorations.analysis_timeseries_queries as esda
+import emission.export.export as eee
+import gzip
+import json
+import bson.json_util as bju
+
+def set_export_data(user_id):
```

Comment:
why is this called `set_export_data`?
We are exporting the data, not just "setting" it, whatever that means.